e2e-tests/utils/auth.util.ts

```diff
@@ -0,0 +1,143 @@
+import { APIRequestContext } from '@playwright/test'
```

Comment:
This can live under `/src/authentication/tests/utils`, no?

Reply:
But everything in here is related to authentication or users, no? (To that point... Some of these could live under the user module and some could live under the authentication model)

You should be able to import from there just as well from where it is now. These tests shouldn't be constrained by the nestjs modular dependency injection like the application code does..

Not blocking.. just seems like a more intuitive place for these IMHO